packages/elements/src/components/table/__styles__/index.tsx

```diff
@@ -123,9 +271,41 @@ export const ElTableRow = styled.div`
   box-shadow: 0px 2px 9px rgba(0, 0, 0, 0.08);
   border-radius: var(--default-border-radius);
 
-  @media screen and (max-width: 750px) {
+  @media only screen and (max-width: 1024px) {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+
+  ${isMobile} {
     grid-template-columns: 1fr 1fr;
   }
+
+  &:hover,
+  &:focus {
+    box-shadow: 0px 0px 3px var(--color-blue-light);
+  }
+`
+
+export const ElTableRowContainer = styled.div`
+  border-radius: var(--default-border-radius);
+
+  &:hover,
+  &:focus {
+    box-shadow: 0px 0px 3px var(--color-blue-light);
+
+    .el-table-row {
+      box-shadow: none;
+    }
+  }
+
+  &${elIsActive} {
```

Comment:
Should that not be `&.${elIsActive}` noting the `.`

Reply:
I tried `&.${elIsActive}` but it didn't work. Plus what I wrote is used elsewhere in the file